CHANGES_NEXT_RELEASE

```diff
@@ -33,6 +33,7 @@ Add:  githash in --version (Issue #1363)
 Add:  Added CLI argument for notification mode. Default mode is 'transient' (previous version used 'permanent' implicitly) (Issue #1370)
 Add:  Added CLI argument for connection memory limit.  Default value is 64Kb (Issue #1384)
 Fix:  Fixed broken subscription cache (Issue #1308), including the semaphore for subscription cache (#1401)
-Fix:  compound attribute values support in CPr update forwardking (Issue #1440)
+Fix:  compound attribute values support in CPr update forwarding (Issue #1440)
 Fix:  Return error response when the (mongo) index key limit is exceeded (Issue #1289)
 Fix:  Incorrect 'details' field rendering (leading to illegal JSON) for error responses in some cases (Issue #843) 
+Fix:  
```

Comment:
That came in there by mistake ...
I was looking for the issue number and finally decided nothing more was to be added.
I only committed CNR as I fixed a spelling error. This line ... by mistake